CHANGES.md

```diff
@@ -1,5 +1,27 @@
 ## unreleased
 
+## v2.5.0
+
+### New features
+* add context reinitialisation (PR #490)
+* add Dolmen frontend (PR #491,#541,#545)
+* add model generation (PR #530)
+* support mutually recursive definitions in the native language (PR #549, #550)
+* support of some options of the SMT-LIB statement (set-option) (PR #608)
+* support for the (get-model) statement (required the Dolmen frontend) (PR #614)
+* support ground reasoning with some bitvector primitives (PR #730, #733, #745).
```

Comment:
Not "some" bitvector primitives -- this should be titled "Support the QF_BV and BV smtlib2 logic" or something similar

Reply:
Done